lib/routes/home/wallet_dashboard.dart

```diff
@@ -130,4 +149,23 @@ class WalletDashboardState extends State<WalletDashboard> {
       },
     );
   }
+
+  void findNextValidFiatConversion(int nextIndex, {int retries = 0}) {
```

Comment:
Maybe a for-loop will read simpler than a recursive function here?

Reply:
https://github.com/breez/breezmobile/pull/338/commits/eb9e3c34ea346715bf0efa1572f535f5428933ec